components/components_CSS/Badge/badge.css

```diff
@@ -0,0 +1,95 @@
+
+@import url("/root.css");
+
+img{
+    max-width: 100%;
+    max-height: 100%;
+    
+ }
+
+ /* -------------------------------------avatar styling */
+.avatar{
+    position: relative;
+
+}
+
+.avater-extra-large{
+   max-height: 15rem;
+    max-width: 15rem;
+
+}
+
+.avatar-large{
+    max-height: 12rem;
+    max-width: 12rem;
+}
+
+.avatar-medium{
+    max-height: 8rem;
+    max-width: 8rem;
+  
+}
+
+.avatar-small{
+    max-height: 5rem;
+    max-width: 5rem;
+}
+
+
+.avatar-extra-small{
+    max-height: 3rem;
+    max-width: 3rem;
+
+}
+
+/* ----------------------------------------Babadge styling */
+.badge{
+    position: absolute;
+    bottom: 0rem;
+    right: 0rem;
+}
+
+.badge-medium{
+    background-color: #59e627;
+    height: 2rem;
+    width: 2rem;
+    border-radius: 15px;
+}
+
+.badge-small{
+    background-color: red;
+    height: 1rem;
+    width: 1rem;
+    border-radius: 15px;
+}
+
+.badge-extra-small{
+    background-color: grey;
+    height: 0.8rem;
+    width: 0.8rem;
+    border-radius: 15px;
+}
```

Comment:
1. you can use short names for classes for short typing like badge extra small can be badge-xl, badge small-badge-sm and so on. 
2. change px/em to rem for responsiveness in less code

Reply:
Thanks Gaurav
1) I thought about it, will try to avoid large names
2) I will fix this